### PR TITLE
Feature/GitHub support

### LINKED ISSUE #25: Fetch document templates directly from github per default
similar to opendevstack/ods-core#518- in case people do NOT want to develop but only use the doc generation service / templates?

should be a pretty simple fix - extend https://github.com/opendevstack/ods-document-generation-svc/blob/master/src/main/groovy/app/DocGen.groovy#L42

### PR DESCRIPTION
**Pull request by clemensutschig**
closes #25

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,5 @@ out/
 
 ### VS Code ###
 .vscode/
+/gradle.properties
```
Comment (metmajer) [inline .gitignore:33]: @clemensutschig is the `/` at the beginning intended?

File: src/test/groovy/app/GithubDocumentTemplatesStoreSpec.groovy [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    def "getRightUrlForVersion"() {
+        given:
+        def store = new GithubDocumentTemplatesStore()
+  
+        when:
+        def url = store.getZipArchiveDownloadURI("1.0")
+  
+        then:
+        "https://www.github.com/opendevstack/ods-document-generation-templates/archive/v1.0.zip".
```
Comment (metmajer) [inline src/test/groovy/app/GithubDocumentTemplatesStoreSpec.groovy:39]: @clemensutschig Groovy allows you to use == for String comparisons. No need to uses `equals()`

**Comment by clemensutschig** [comment]
Still wip.. github returns a different File Structure..  need to play with this tmrw..

File: .gitignore (hunk 1 above)
Comment (clemensutschig) [reply .gitignore:33]: works :)

**Comment by clemensutschig** [comment]
@metmajer  - hurray, finally ... any furhter comments, if not - I'll merge

**Pull request merged**